### PR TITLE
feat(makefile): Update makefile

### PR DESCRIPTION
**Pull request by flooie**
Add multiarch development push
Simplify development image builds

@mlissner I think it makes sense to add a multi arch push for development images, but 
I'm hesitant to just include it in the normal process for pushing builds for the server. 

I also dropped the tagging of web dev versions because ... who is using those anyway.

### CODE REVIEW

**Comment by flooie** [comment]
@mlissner only the web-dev was being built and not pushed right?  I think we should build a celery-dev too agreed?

**Comment by mlissner** [comment]
I guess if we are running the `push` command, it should build everything that it tries to push or else the push is buggy?

File: .github/workflows/docker-build-dev.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Automate build and deploy
```
Comment (mlissner) [inline .github/workflows/docker-build-dev.yml:1]:
```suggestion
name: Push new dev images
```

File: .github/workflows/docker-build-dev.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and Push
+        run: |
+          make multiarch_push_development --file docker/django/Makefile
```
Comment (mlissner) [inline .github/workflows/docker-build-dev.yml:22]:
```suggestion
          make multiarch_push_development --file docker/django/Makefile  -e VERSION=$(git rev-parse --short HEAD)
```

**Review by mlissner — CHANGES_REQUESTED** [review]

**Review by mlissner — APPROVED** [review]

**Pull request merged**